content/en/docs/ops/best-practices/security/index.md

```diff
@@ -90,6 +90,80 @@ spec:
         notPaths: ["/public"]
 {{< /text >}}
 
+### Understand path normalization in authorization policy
+
+The enforcement point of authorization policies is the Envoy proxy which is different from the actual resource access point that is
+usually in the backend application. A policy mismatch happens when the Envoy proxy and the backend application interprets the request
```

Comment:
```suggestion
The enforcement point for authorization policies is the Envoy proxy instead of the usual resource access point in the backend application. A policy mismatch happens when the Envoy proxy and the backend application interpret the request
```